apps/twap/src/app/orderbook/trade.ts

```diff
@@ -0,0 +1,31 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import {
+  bigIntToString,
+  bufferToString,
+  stringToBigInt,
+  stringToBuffer,
+} from '../utils/transformers';
+
+@Entity({ name: 'trades' })
+export class Trade {
+  @PrimaryColumn('bigint', {
+    name: 'block_number',
+    transformer: { from: stringToBigInt, to: bigIntToString },
+  })
+  blockNumber: bigint;
+
+  @PrimaryColumn('bigint', {
+    name: 'log_index',
+    transformer: { from: stringToBigInt, to: bigIntToString },
+  })
+  logIndex: bigint;
+
+  @Column('bytea', {
+    name: 'order_uid',
+    transformer: { from: bufferToString, to: stringToBuffer },
+  })
+  orderUid: string;
+
+  @Column('numeric', { name: 'sell_amount' })
+  sellAmount: string;
+}
```

Comment:
Is there no buy amount as part of the trade?

Reply:
![image](https://github.com/cowprotocol/cow-web-services/assets/5172699/e751edf3-b45b-494d-8c00-ba92228ba064)
I must have missed it, fixing it.